.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @frol
\ No newline at end of file
+* @Shelnutt2 @frol @ihnorton
```

Comment:
I am no longer heavily involved in conda or capnproto

```suggestion
* @Shelnutt2 @ihnorton
```

Reply:
@frol this change needs to be made in the recipe and then re-render will update this. Otherwise it will be added back in the next re-render